Documentation/doc/Documentation/Developer_manual/Chapter_portability.txt

```diff
@@ -304,7 +304,7 @@ up-to-date version of this list.
 \subsection secworkaround_macros Macros connected to workarounds/compilers
 
 Some macros are defined according to certain workaround flags. This is
-done to avoid some \#`ifdef`s in our actual code.
+done to avoid some `#ifdef`s in our actual code.
```

Comment:
```suggestion
done to avoid some `ifdef`s in our actual code.
```

Reply:
This is also good suggestion and will probably not really disturb the readability.